nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
```

Comment:
So it looks like the C++ stores `stats` internally as a map while the Python stores it as an array. We should probably figure out which one makes sense and standardize it. I think the enumeration is fine for now.